app/res/layout/screen_login.xml

```diff
@@ -1,8 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent" 
-    android:id="@+id/screen_login_main">
+    android:layout_height="match_parent"
+    android:id="@+id/screen_login_main">
+    
+    <FrameLayout
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
+        android:paddingBottom="100sp"

```

Comment:
should be in the dimens file (if we can't avoid it entirely). Also: does this scale properly on different devices?